### ISSUE TITLE
The closing <span/> tag is causing a warning in Vite.

### ISSUE THREAD

**Issue by Takuro1000** [ghostdevv/svelte-hamburgers#154]
## Overview

The closing `span` tag is causing a warning in Vite.

## Details

A warning is being issued by Vite with the following code.

```bash
vite v5.3.5 building SSR bundle for production...
transforming (43) node_modules/@sveltejs/kit/src/runtime/server/page/respond_with_error.js
[vite-plugin-svelte] node_modules/svelte-hamburgers/dist/Hamburger.svelte:32:8 Self-closing HTML tags for non-void elements are ambiguous — use `<span ...></span>` rather than `<span ... />`
30:     on:click={() => (open = !open)}>
31:     <span class="hamburger-box">
32:         <span class="hamburger-inner" />
                                            ^
33:     </span>
34: </button>

```

## Expected Behavior

To prevent the warning, `<span ...></span>` should be used instead of `<span ... />`.

## Environment

- Vite: v5.3.5
- Svelte: 5.0.0-next.203
- svelte-hamburgers: 4.2.0

## Reference Links

[[Relevant File](https://github.com/ghostdevv/svelte-hamburgers/blob/main/src/lib/Hamburger.svelte)](https://github.com/ghostdevv/svelte-hamburgers/blob/main/src/lib/Hamburger.svelte)

**Comment by ghostdevv** [comment]
Ah good catch! It's quite an easy fix, to just change the lines in the file here: https://github.com/ghostdevv/svelte-hamburgers/blob/854c9c573cb0cff44a6ce38abe28bf68572d7006/src/lib/Hamburger.svelte#L36

Would you like to make a PR @Takuro1000 ? If not I can take a look at this :pray: /nf

**Comment by Takuro1000** [comment]
I have just submitted a pull request🙇

**Comment by ghostdevv** [comment]
Thanks! @Takuro1000 Released in `v4.2.1`
